app/views/accounts/accountables/investment/_tabs.html.erb

```diff
@@ -1,28 +1,34 @@
 <%# locals: (account:, selected_tab:) %>
 
-<% if account.entries.account_trades.any? || account.entries.account_transactions.any? %>
+<% if account.mode.nil? %>
+  <%= render "accounts/accountables/value_onboarding", account: account %>
+<% else %>
   <div class="flex gap-2 text-sm text-gray-900 font-medium mb-4">
-    <%= render "accounts/accountables/tab", account: account, key: "holdings", is_selected: selected_tab.in?([nil, "holdings"]) %>
-    <%= render "accounts/accountables/tab", account: account, key: "cash", is_selected: selected_tab == "cash" %>
-    <%= render "accounts/accountables/tab", account: account, key: "transactions", is_selected: selected_tab == "transactions" %>
+    <% if account.mode == "transactions" %>
+      <%= render "accounts/accountables/tab", account: account, key: "holdings", is_selected: selected_tab.in?([nil, "holdings"]) %>
+      <%= render "accounts/accountables/tab", account: account, key: "cash", is_selected: selected_tab == "cash" %>
+      <%= render "accounts/accountables/tab", account: account, key: "transactions", is_selected: selected_tab == "transactions" %>
+    <% end %>
   </div>
 
   <div class="min-h-[800px]">
-    <% case selected_tab %>
-    <% when nil, "holdings" %>
-      <%= turbo_frame_tag dom_id(account, :holdings), src: account_holdings_path(account) do %>
-        <%= render "account/entries/loading" %>
-      <% end %>
-    <% when "cash" %>
-      <%= turbo_frame_tag dom_id(account, :cash), src: account_cashes_path(account) do %>
-        <%= render "account/entries/loading" %>
-      <% end %>
-    <% when "transactions" %>
-      <%= turbo_frame_tag dom_id(account, :trades), src: account_trades_path(account) do %>
-        <%= render "account/entries/loading" %>
+    <% if account.mode == "transactions" %>
+      <% case selected_tab %>
+      <% when nil, "holdings" %>
+        <%= turbo_frame_tag dom_id(account, :holdings), src: account_holdings_path(account) do %>
+          <%= render "account/entries/loading" %>
+        <% end %>
+      <% when "cash" %>
+        <%= turbo_frame_tag dom_id(account, :cash), src: account_cashes_path(account) do %>
+          <%= render "account/entries/loading" %>
+        <% end %>
+      <% when "transactions" %>
+        <%= turbo_frame_tag dom_id(account, :trades), src: account_trades_path(account) do %>
+          <%= render "account/entries/loading" %>
+        <% end %>
       <% end %>
+    <% else %>
+      <%= render "accounts/accountables/valuations", account: account %>
     <% end %>
```

Comment:
Will need to come back around and clean a lot of this markup up.  Lots of duplication across the accountable views